datamodels/2.x/itop-core-update/dictionaries/pl.dict.itop-core-update.php

```diff
@@ -0,0 +1,118 @@
+	'itop-core-update/Operation:ConfirmUpdate/Title' => 'Potwierdź aktualizację aplikacji',
+	'itop-core-update/Operation:UpdateCoreFiles/Title' => 'Aktualizacja aplikacji',
+	'itop-core-update/Operation:UpdateDone/Title' => 'Aktualizacja aplikacji zakończona',
+
+	'iTopUpdate:UI:SelectUpdateFile' => 'Wybierz plik aktualizacji do przesłania',
+	'iTopUpdate:UI:CheckUpdate' => 'Weryfikacja pliku aktualizacji',
+	'iTopUpdate:UI:ConfirmInstallFile' => 'Masz zamiar zainstalować %1$s',
+	'iTopUpdate:UI:DoUpdate' => 'Aktualizacja',
+	'iTopUpdate:UI:CurrentVersion' => 'Aktualnie zainstalowana wersja',
+	'iTopUpdate:UI:NewVersion' => 'Nowo zainstalowana wersja',
+    'iTopUpdate:UI:Back' => 'Wstecz',
+    'iTopUpdate:UI:Cancel' => 'Anuluj',
+    'iTopUpdate:UI:Continue' => 'Kontynuuj',
+	'iTopUpdate:UI:RunSetup' => 'Uruchom instalację',
+    'iTopUpdate:UI:WithDBBackup' => 'Kopia zapasowa bazy danych',
+    'iTopUpdate:UI:WithFilesBackup' => 'Kopia zapasowa plików aplikacji',
+    'iTopUpdate:UI:WithoutBackup' => 'Nie ma zaplanowanych kopii zapasowych',
+    'iTopUpdate:UI:Backup' => 'Kopia zapasowa wygenerowana przed aktualizacją',
+	'iTopUpdate:UI:DoFilesArchive' => 'Archiwizuj pliki aplikacji',
+	'iTopUpdate:UI:UploadArchive' => 'Wybierz pakiet do przesłania',
+	'iTopUpdate:UI:ServerFile' => 'Ścieżka pakietu znajdującego się na serwerze',
+	'iTopUpdate:UI:WarningReadOnlyDuringUpdate' => 'Podczas aktualizacji aplikacja będzie tylko do odczytu.',
+
+    'iTopUpdate:UI:Status' => 'Status',
+    'iTopUpdate:UI:Action' => 'Aktualizacja',
+    'iTopUpdate:UI:History' => 'Historia wersji',
+    'iTopUpdate:UI:Progress' => 'Progress of the upgrade',
+
+    'iTopUpdate:UI:DoBackup:Label' => 'Kopie zapasowe plików i bazy danych',
+    'iTopUpdate:UI:DoBackup:Warning' => 'Tworzenie kopii zapasowych nie jest zalecane ze względu na ograniczoną ilość wolnego miejsca na dysku',
+
+    'iTopUpdate:UI:DiskFreeSpace' => 'Wolne miejsce na dysku',
+    'iTopUpdate:UI:ItopDiskSpace' => 'Przestrzeń dyskowa  iTop',
+    'iTopUpdate:UI:DBDiskSpace' => 'Przestrzeń dyskowa bazy danych',
+	'iTopUpdate:UI:FileUploadMaxSize' => 'Maksymalny rozmiar przesyłanego pliku',
+
+	'iTopUpdate:UI:PostMaxSize' => 'Wartość PHP ini post_max_size: %1$s',
+	'iTopUpdate:UI:UploadMaxFileSize' => 'Wartość PHP ini upload_max_filesize: %1$s',
+
+    'iTopUpdate:UI:CanCoreUpdate:Loading' => 'Sprawdzanie plików',
+    'iTopUpdate:UI:CanCoreUpdate:Error' => 'Sprawdzanie plików nie powiodło się (%1$s)',
+    'iTopUpdate:UI:CanCoreUpdate:ErrorFileNotExist' => 'Sprawdzanie plików nie powiodło się (plik nie istnieje %1$s)',
+    'iTopUpdate:UI:CanCoreUpdate:Failed' => 'Sprawdzanie plików nie powiodło się',
+    'iTopUpdate:UI:CanCoreUpdate:Yes' => 'Aplikacja może być zaktualizowana',
+	'iTopUpdate:UI:CanCoreUpdate:No' => 'Nie można zaktualizować aplikacji: %1$s',
+	'iTopUpdate:UI:CanCoreUpdate:Warning' => 'Ostrzeżenie: aktualizacja aplikacji może się nie powieść: %1$s',
+

```

Comment:
Comparing with the EN file, missing this key : 

```suggestion
	'iTopUpdate:UI:CannotUpdateUseSetup' => 'You must use the <a href="%1$s">setup</a> to update the application.<br />Some modified files were detected, a partial update cannot be executed.~~',


```

Reply:
Done